sonar-cxx-plugin/src/main/java/org/sonar/plugins/cxx/coverage/CxxCoverageSensor.java

```diff
@@ -165,7 +165,9 @@ private void saveMeasures(SensorContext context,
       String filePath = entry.getKey();
       InputFile cxxFile = fs.inputFile(fs.predicates().hasPath(filePath));
       if (cxxFile != null) {
-        Collection<Measure> measures = entry.getValue().createMeasures();
+        CxxUtils.LOG.debug("Saving for file '{}'", filePath);
+        CoverageMeasuresBuilder builder = entry.getValue();
```

Comment:
I had the case where an exception was thrown 
entry.getValue().createMeasures();
and it is easier to analyze if there is no chained method call